bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Navigation.java

```diff
@@ -82,4 +82,14 @@ default String getExportedType() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Retrieves the value of label for navigation.
```

Comment:
We should mention the accessibility intention of the label here

Reply:
Would "Retrieves the value of aria-label for navigation. Improves accessibility of component" sound better?

Reply:
Changed all the Javadocs, let me know if the new ones are ok

Reply:
I probably wouldn't mention the `aria-label` attribute here, it's an implementation detail, you could equally use `aria-labelledby`, for example. Maybe: "Retrieves an accessible label that describes the navigation." is better?

Reply:
Done